packages/conversation/src/rest/v1/conversation-domain-api.ts

```diff
@@ -83,7 +83,8 @@ export class ConversationDomainApi implements Api {
       }
       const apiClientOptions = this.buildApiClientOptions(this.sinchClientParameters);
       this.client = new ApiFetchClient(apiClientOptions);
-      this.client.apiClientOptions.basePath = this.buildBasePath(region);
+      this.client.apiClientOptions.hostname = this.sinchClientParameters.conversationHostname
```

Comment:
Another point: may be I missed something from logic but seems you cannot set 2 different hostname to be used for the 2 supported endpoint: `conversation` and `template` endpoints.
You can only override one at same time ?

Reply:
Indeed, this is a miss from my end. I'll do it for voice application management too

Reply:
Fixed 🙂